Tests/SwiftDocCTests/Diagnostics/DiagnosticTests.swift

```diff
@@ -140,6 +141,52 @@ class DiagnosticTests: XCTestCase {
         \(expectedNoteLocation): note: The message of the note.
         """)
     }
+
+    func createTestSymbol(commentText: String) -> SymbolGraph.Symbol {
+         let emptyRange = SymbolGraph.LineList.SourceRange(
+             start: .init(line: 0, character: 0),
+             end: .init(line: 0, character: 0)
+         )
+        let docCommentLines = commentText.components(separatedBy: .newlines).map { SymbolGraph.LineList.Line(text: $0, range: emptyRange) }
+        return SymbolGraph.Symbol(
+            identifier: .init(precise: "s:5MyKit0A5ClassC10myFunctionyyF", interfaceLanguage: "occ"),
+            names: .init(title: "", navigator: nil, subHeading: nil, prose: nil),
+            pathComponents: ["path", "to", "my file.h"],
+            docComment: SymbolGraph.LineList(docCommentLines),
+            accessLevel: .init(rawValue: "public"),
+            kind: .init(parsedIdentifier: .func, displayName: "myFunction"),
+            mixins: [:]
+        )
+    }
+
+    func testDoxygenDiagnostic() throws {
+        let commentText = """
+          Brief description of this method
+          @param something Description of this parameter
+          @returns Description of return value
+          """
+        let symbol = createTestSymbol(commentText: commentText)
+        let engine = DiagnosticEngine()
+
+        let _ = DocumentationNode.contentFrom(documentedSymbol: symbol, documentationExtension: nil, engine: engine)
+        XCTAssertEqual(engine.problems.count, 0)
+
+        // testing scenario with known directive
+        let commentWithKnownDirective = """
+          /// Brief description of this method
+          @Image something Description of this parameter
```

Comment:
Non-blocking but it might be nice to update this to something we'd more expect to see in code: https://www.swift.org/documentation/docc/image

```suggestion
          Brief description of this method
          
          @Image(source: "my-sloth-image.png", alt: "An illustration of a sleeping sloth.")
```